### DISCUSSION TITLE
Button Type - Scene. Current Button Type implementation doesn't allow activation of scenes

### DISCUSSION THREAD

**Discussion by prestonsmartuk** [Clooos/Bubble-Card#400] [Feature requests]
I tried using the Button Type 'Switch' to use a Scene entity (Movie mode lights, chill lights etc), but it doesn't activate the lights scene. It would be good if the Button Type was expanded to include 'Scenes'

**Comment by Clooos** [comment]
Thanks for your feedback. For now you can do that with the custom button. Just take a look in the documentation, this is explained here.
